src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -38,7 +38,8 @@
         .bcs-activity-feed-task,
         .bcs-activity-feed-comment {
             border-bottom: 1px solid $ffive;
-            padding-bottom: 8px;
+            margin-top: 20px;
```

Comment:
Are these CSS changes necessary to support the order change?

Reply:
I can move this into a separate PR
@jstoffan